lib/active_model/serializer.rb

```diff
@@ -73,7 +73,7 @@ def self.serializer_lookup_chain_for(klass, namespace = nil)
     # Used to cache serializer name => serializer class
     # when looked up by Serializer.get_serializer_for.
     def self.serializers_cache
-      @serializers_cache ||= ThreadSafe::Cache.new
+      @serializers_cache ||= Concurrent::Map.new
```

Comment:
kind of odd.  I'm not sure where the dependency is coming from.  I know later versions of Rails introduce concurrent-ruby.  I guess thread_safe was coming from active_support?

Reply:
`thread_safe` is needed by 2 different sources:
- `active_support`
- `grape` (through `virtus`)